mcploader.js

```
@@ -0,0 +1,126 @@
+                            "isError": isError
+                        };
+                    }
+                    
+                    if (function_calling_name == "evaluate_js_es3") {
+                        return {
+                            "content": [
+                                {
+                                    "type": "text",
+                                    "text": (function(script) {
+                                        try {
+                                            return String(eval(script));
```

Biome (2.4.10)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)